docs/origin_install.md

```diff
@@ -0,0 +1,154 @@
+        The directories specified this way are writable by default.
+        Access controls should be managed via [authorization configuration](#configuring-authentication-and-authorization).
+
+1. In `/etc/xrootd/config.d/10-common-site-local.cfg`, add a line to set the `resourcename` variable.
+   Unless your supported VOs' policies state otherwise,
+   this should match the [resource name](../../common/registration.md#registering-resources) of your XRootD service.
+   For example, the XRootD service registered at the
+   [University of Florida site](https://github.com/opensciencegrid/topology/blob/b14218d6e9d9df013a42e4d8538b2eeea615514c/topology/University%20of%20Florida/UF%20HPC/UFlorida-HPC.yaml#L250)
+   should set the following configuration:
+
+        set resourcename = UFlorida-XRD
+
+   For example, the XRootD service registered at the
+   [University of Florida site](https://github.com/opensciencegrid/topology/blob/b14218d6e9d9df013a42e4d8538b2eeea615514c/topology/University%20of%20Florida/UF%20HPC/UFlorida-HPC.yaml#L250)
+   should set the following configuration:
+
+        set resourcename = UFlorida-XRD
+
+### Configuring authentication and authorization
+
+XRootD offers several authentication options using security plugins to validate incoming credentials, such as bearer tokens, X.509 proxies, and VOMS proxies.
+Please follow the [XRootD authorization documentation](xrootd-authorization.md) for instructions on how to configure authentication and authorization,
+including validating credentials and mapping them to users if desired.
+
+### Optional configuration
+
+The following configuration steps are optional and will likely not be required for setting up a small site.
+If you do not need any of the following special configurations, skip to
+[the section on using XRootD](#using-xrootd).
+
+
+#### Enabling multi-user support
+
+!!! warning "Requirements for XRootD-Multiuser with VOMS FQANs"
+    Using XRootD-Multiuser with a VOMS FQAN requires mapping the FQAN to a username, which requires a `voms-mapfile`.
+    Support is available in `xrootd-voms 5.4.2-1.1`, in the OSG 3.6 repos, though it is expected in XRootD 5.5.0.
+    If you want to use multiuser, ensure you are getting `xrootd-voms` from the OSG repos.
+
+The `xrootd-multiuser` plugin allows XRootD to write files on the storage system as the
+[authenticated](xrootd-authorization.md) user instead of the `xrootd` user.
+If your XRootD service only allows read-only access, you should skip installation of this plugin.
+
+To set up XRootD in multi-user mode, install the `xrootd-multiuser` package:
+
+``` console
+root@xrootd-standalone # yum install xrootd-multiuser
+```
+
+Installing Pelican
+------------------
+
+Grab the appropriate binary for your system from the [pelican repository](https://github.com/PelicanPlatform/pelican/releases)
+
+Launching an Origin
+-------------------
+
+To launch a pelican origin, run:
+
+```./pelican origin -f <federation> -v <volume>```
```

Comment:
This is missing the `serve` sub-command, and the fact that the argument to the volume flag takes two pieces. We need ```pelican origin serve -f <federation url> -v <local directory>:<namespace prefix>```